handlers/base.js

```diff
@@ -9,7 +9,11 @@ const getClassComment = (item, s, proto, commentPos, requirements) => {
 ${normalizeComments(findCommentByPath([commentPos, s], proto.sourceCodeInfo.locationList), 1)}
  * ${item.name} class generated from protobuf definition "${proto.name}".
  * auto-generated code PLEASE DO NOT EDIT!
- * ${requirements ? requirements.join('\n * ') : ''}
+ *
+ * @ignore grpc
+ * @ignore jspb
+ * @ignore proto.google.protobuf
```

Comment:
You can add all of them in one line like `@ignore(grpc, jspb, proto.google.protobuf)` see https://qooxdoo.org/documentation/v7.0/#/development/apidoc/api_jsdoc_at_ignore